js/components/Checkbox.js

```diff
@@ -0,0 +1,133 @@
+'use strict';
```

Comment:
I only upgraded that Checkbox module to support the new react-native versions. It would take too long to keep sending commits back and forth to your fork, given the amount of work that had to be done to bump our codebase to the new requirements.

If you'd like to update your repo  with those changes I made, that code should work as a module as well. :)

Let me know which approach you'd prefer for this case.